dataflow_pipeline.py

```diff
@@ -0,0 +1,111 @@
+from __future__ import absolute_import
+
+import argparse
+import logging
+import re
+import timeit
+
+import apache_beam as beam
+from apache_beam.io import ReadFromText
+from apache_beam.io import WriteToText
+from apache_beam.metrics import Metrics
+from apache_beam.metrics.metric import MetricsFilter
+from apache_beam.options.pipeline_options import PipelineOptions
+from apache_beam.options.pipeline_options import SetupOptions
+import json
+
+from scripts.google_replicate import exec_google_copy
+
+try:
```

Comment:
why is this necessary? can you add some comments explaining this? I assume it's some Python 2/3 compatibility?